inst/rmarkdown/templates/trundler_template/skeleton/skeleton.Rmd

```diff
@@ -157,23 +215,32 @@ pineapple_summary <- complete(pineapple_consolidated, date = dates, nesting(sku)
   mutate(price_interpolate = approx(date, price, date, method = "constant", rule = 2)$y) %>%
```

Comment:
This got me thinking. Should we build interpolation functionality into the package directly?